arcade/arcade/cli/new.py

```diff
@@ -102,147 +101,28 @@ def create_new_toolkit(directory: str) -> None:
                 "Please try again.[/red]"
             )
 
-    description = ask_question("Description of the toolkit?")
-    author_name = ask_question("Author's name?")
-    author_email = ask_question("Author's email?")
-    author = f"{author_name} <{author_email}>"
-
-    yes_options = ["yes", "y", "ye", "yea", "yeah", "true"]
-    generate_test_dir = (
-        ask_question("Generate test directory? (yes/no)", "yes").lower() in yes_options
+    toolkit_description = ask_question("Description of the toolkit?")
+    toolkit_author_name = ask_question("Github owner username?")
+    toolkit_author_email = ask_question("Author's email?")
+
+    context = {
+        "package_name": package_name,
+        "toolkit_name": toolkit_name,
+        "toolkit_description": toolkit_description,
+        "toolkit_author_name": toolkit_author_name,
+        "toolkit_author_email": toolkit_author_email,
+        "arcade_version": f"{ARCADE_VERSION.rsplit('.', 1)[0]}.*",
```

Comment:
nit, but we are not validating this anywhere and just trusting that it has a `.`